dynawo/sources/Models/Modelica/Dynawo/Electrical/Loads/LoadAlphaBetaRestorative.mo

```diff
@@ -33,10 +33,19 @@ model LoadAlphaBetaRestorative "Generic model of a restorative Alpha-Beta load."
 
   equation
     if (running.value) then
-      tFilter * der(UFilteredRawPu) = ComplexMath.'abs' (terminal.V) - UFilteredRawPu;
+
       UFilteredPu = if UFilteredRawPu >= UMaxPu then UMaxPu elseif UFilteredRawPu <= UMinPu then UMinPu else UFilteredRawPu;
-      PPu = PRefPu.value * ((ComplexMath.'abs' (terminal.V) / UFilteredPu) ^ Alpha);
-      QPu = QRefPu.value * ((ComplexMath.'abs' (terminal.V) / UFilteredPu) ^ Beta);
+
+      if (ComplexMath.'abs' (terminal.V)==0) then
+        PPu=0;
+        QPu=0;
+        UFilteredRawPu=0;
+      else
+        tFilter * der(UFilteredRawPu) = ComplexMath.'abs' (terminal.V) - UFilteredRawPu;
```

Comment:
In fact I think we can keep this equation outside the if (just before the UfilteredPu = if UFilteredRawPu >=...). This way we only keep PPu and QPu calculation in the if, which are the only variables concerned by the division.